lib/src/airship_flutter.dart

```diff
@@ -250,6 +250,10 @@ class Airship {
     return await _channel.invokeMethod('deleteInboxMessage', message.messageId);
   }
 
+  static Future<void> refreshInbox() async {
```

Comment:
can be updated to Future<bool> if you do the chaining right with the completion handlers